trame_rca/utils.py

```diff
@@ -275,6 +276,8 @@ def update_size(self, origin, size):
         width = max(1, int(size.get("w", 300)))
         height = max(1, int(size.get("h", 300)))
         self._iren.UpdateSize(width, height)
+        # FIXME: see https://gitlab.kitware.com/vtk/vtk/-/merge_requests/11905
```

Comment:
Add something about removing that block for vtk 9.5+ thanks to MR#11905

You could even test vtk version and conditionally trigger that event.

BTW, thanks for fixing the VTK/C++ side!

Reply:
My worry is that we won't know what to do later on. So let's write the next step in the comment so we don't have to think about it down the road. ;-)